sysl2/sysl/seqs.go

```diff
@@ -237,6 +239,31 @@ func DoGenerateSequenceDiagrams(args []string) error {
 			"/project_dir/my_models combine with --root if needed",
 	).String()
 
+	flags = append(flags, "root", "endpoint_format", "app_format", "title", "plantuml",
```

Comment:
this validation should be required for all commands.

Reply:
Done.